chat-plugins/trivia.js

```diff
@@ -418,27 +406,33 @@ var commands = {
 	// trivia game commands
 	new: function (target, room) {
 		if (room.id !== 'trivia' || !this.can('broadcast', null, room) || !target) return false;
-		Trivia.getInstance().startSignups(target, room, this);
+		if (!room.Trivia) room.Trivia = Trivia;
```

Comment:
The reason I didn't attach it to ``CommandParser`` is because it's still undefined when the script is first started, but I hadn't thought to wait until ``getInstance`` is first called to attach them.